pubspec.yaml

```
@@ -1,5 +1,5 @@
 name: code_builder
-version: 4.2.0-dev
+version: 4.2.1-dev
```

Looks like we were already on a dev version so you can keep the number the same

```suggestion
version: 4.2.0-dev
```